packages/backend-plugin-api/src/services/definitions/HttpRouterService.ts

```diff
@@ -22,11 +22,16 @@ export interface HttpRouterServiceAuthPolicy {
   allow: 'unauthenticated' | 'user-cookie';
 }
 
+/** @public */
+export interface HttpRouterHealthCheckConfig {
+  handler: () => Promise<any>;
+}
+
 /**
  * @public
  */
 export interface HttpRouterService {
   use(handler: Handler): void;
-
+  healthCheckConfig(healthCheckOptions: HttpRouterHealthCheckConfig): void;
```

Comment:
It would be nice to add documentation about this new configuration on the `HttpRouterService` page:
https://backstage.io/docs/backend-system/core-services/http-router#configuring-the-service